### DISCUSSION TITLE
redhat 8.9 service registration issue (start boot on)

### DISCUSSION THREAD

**Discussion by WonhoiKim** [bluenviron/mediamtx#3163] [Questions]
### Question

HI I have been using this program well since rtsp-simple-server. When I tried to register this program as a service in redhat 8.9, an error occurred.

I entered the command on the homepage. (https://github.com/bluenviron/mediamtx?tab=readme-ov-file#linux)

Commands that are possible in the Ubuntu 20.04 fail in Redhat 8.9. The service error details are as follows.

`
[root@localhost etc]# systemctl status mediamtx.service 
● mediamtx.service
   Loaded: loaded (/etc/systemd/system/mediamtx.service; disabled; vendor preset: disabled)
   Active: failed (Result: exit-code) since Sun 2024-03-24 12:03:48 EDT; 8min ago
 Main PID: 6789 (code=exited, status=203/EXEC)

 3월 24 12:03:48 localhost.localdomain systemd[1]: Started mediamtx.service.
 3월 24 12:03:48 localhost.localdomain systemd[1]: mediamtx.service: Main process exited, code=exited, status=203/EXEC
 3월 24 12:03:48 localhost.localdomain systemd[1]: mediamtx.service: Failed with result 'exit-code'.
`

The mediaxmtx executable runs normally on the console. There seems to be an issue with the service not being registered.
I would like to know how to resolve this error. please reply...

Thanks

**Comment by aler9** [comment]
Hello, instructions reported in the README are for Ubuntu only. In case of other distributions, you need to dig into to the documentation of that specific distribution and find out how services can be created there.

There are plenty of answers on how to fix that error on Red Hat, this is not a server-related question but a Red Hat-related question.

[a comment by an automated account is not shown]
